torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -299,21 +198,43 @@ void initNvFuserPythonBindings(PyObject* module) {
             }
           },
           py::arg("dtype") = torch::jit::fuser::cuda::DataType::Double,
-          py::return_value_policy::reference);
+          py::return_value_policy::reference); */
 
-  py::class_<FusionDefinitionContextManager::Ops> nvf_ops(fusion_def, "Ops");
+  py::class_<nvfuser::FusionDefinition::Operators> nvf_ops(fusion_def, "Operators");
+  nvf_ops.def(py::init<nvfuser::FusionDefinition*>());
 
   // ******************** INSERT OP BINDINGS BELOW HERE ********************
 
-#define NVFUSER_PYTHON_BINDING_UNARY_OP(op_str, op_name)                 \
-  nvf_ops.def_static(                                                    \
-      op_str,                                                            \
-      py::overload_cast<TensorView*>(&torch::jit::fuser::cuda::op_name), \
-      py::return_value_policy::reference);                               \
-  nvf_ops.def_static(                                                    \
-      op_str,                                                            \
-      py::overload_cast<torch::jit::fuser::cuda::Val*>(                  \
-          &torch::jit::fuser::cuda::op_name),                            \
+#define NVFUSER_PYTHON_BINDING_UNARY_OP(op_str, op_name)                       \
+  nvf_ops.def(                                                                 \
+      op_str,                                                                  \
+      [](nvfuser::FusionDefinition::Operators& self, nvfuser::Tensor* input)   \
+         -> nvfuser::Tensor* {                                                 \
+        nvfuser::Tensor* output =                                              \
+          new nvfuser::Tensor(self.fusion_definition->recording_state.size()); \
+        self.fusion_definition->recording_state.emplace_back(output);          \
+        self.fusion_definition->recording.emplace_back(                        \
+          new nvfuser::OpRecord<NvfTensorView*, NvfTensorView*>(               \
```

Comment:
nitpick: we can probably use typedef/aliases here for `nvfuser::OpRecord<xxxxxxx>`, which would make code easier to read.
something like `using unary_tv = nvfuser::OpRecord<NvfTensorView*, NvfTensorView*>`

Reply:
I wasn't sure this would make much difference since `nvfuser::OpRecord<NvfTensorView*, NvfTensorView*>` is not reused and you have to check somewhere else to understand the shorthand.